src/raster/r.random.walk/r.random.walk.html

```diff
@@ -0,0 +1,36 @@
+<h2>DESCRIPTION</h2>
+The <em>r.random.walk</em> module generates a 2D random walk across the current computational region.
+The module provides control of the number of steps and directions (4 or 8) a walker can take and allows the walker's behavior to be set to be self-avoiding (Madras et al., 1996) or allow revisits. The output displays the frequency the walker visited each cell or the average frequency. The module can run multiple walks in parallel. It either samples the same starting location for each walk or generates a unique starting position for each walker.
```

Comment:
```suggestion
The module provides control of the number of steps and directions (4 or 8) a walker can take and allows
the walker's behavior to be set to be self-avoiding (Madras et al., 1996) or allow revisits. The output displays
the frequency the walker visited each cell or the average frequency. The module can run multiple walks in
parallel. It either samples the same starting location for each walk or generates a unique starting position
for each walker.
```

(long line broken, according to https://trac.osgeo.org/grass/wiki/Submitting/Docs#Breaklonglines)